app/main.py

```diff
@@ -2,0 +1,22 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if parts[2].endswith("/"):
+        new_path = os.path.join(parts[2], os.path.basename(parts[1]))
+    else:
+        new_path = parts[2]
+
+    directory = os.path.dirname(new_path)
+    if directory and not os.path.exists(directory):
+        new_dir = ""
+        for part in directory.split("/"):
+            new_dir = os.path.join(new_dir, part)
+            if not os.path.exists(new_dir):
+                if not os.path.isfile(new_dir):
+                    os.mkdir(new_dir)
+
+    if os.path.exists(parts[1]):
```

Comment:
It's good that you check if the source file exists before renaming. However, you might want to add error handling (e.g., try-except block) around the `os.rename` operation to catch any potential exceptions that could occur during the file move, such as permission errors.